send/generic_sender.py

```diff
@@ -83,6 +83,8 @@ def prepare_url_transport_command(temp, content_type: str = "application/x-tar",
 	# add standard CURL params
 	transport_command.extend(["-i", "-H", "Content-Type:" + content_type, "-w", "%{http_code}", "--show-error",
 							  "--silent", "-o", temp.name, "-X", method, "--data-binary", "@-"])
+	if auth_token:
+		transport_command.extend(["-H", "Authorization: Bearer " + auth_token])
```

Comment:
Based on the original docs, they are expecting Authorization header just with the token value (no `Bearer`). At least it worked that way when I originally tested the API. I'll check with the service provider.

Note: Just because they are using "tokens" it doesn't have to mean they are implementing OAuth 2.0, which bearer authorization scheme is a part of. But I would prefer to use the standard, so I'll talk to them.

Reply:
They updated the enpoint to expect Bearer tokens, just the verification fails for now.